layouts/shortcodes/pages/release/10.html

```diff
@@ -0,0 +1,158 @@
+  <a class="jakartaee-9-platform-img-container margin-top-30" href="/release/10/images/jakarta-ee-10-platform.jpg">
+    <img class="jakartaee-platform-img" src="/release/10/images/jakarta-ee-10-platform.jpg" alt="Jakarta EE 10 platform">
+  </a>
+  <div class="row">
+    <div class="col-sm-18 col-sm-offset-3">
+      <p class="text-center big margin-top-60 margin-bottom-20">{{ i18n "jakarta-ee-version-10-specifications-text"  | safeHTML }}</p>
+    </div>
+    <div class="col-sm-12 col-sm-offset-6">
+      <ul>
+        <li>{{ i18n "jakarta-ee-version-10-specifications-list1"  | safeHTML }}</li>
+        <li>{{ i18n "jakarta-ee-version-10-specifications-list2"  | safeHTML }}</li>
+        <li>{{ i18n "jakarta-ee-version-10-specifications-list3"  | safeHTML }}</li>
+        <li>{{ i18n "jakarta-ee-version-10-specifications-list4"  | safeHTML }}</li>
+        <li>{{ i18n "jakarta-ee-version-10-specifications-list5"  | safeHTML }}</li>
+        <li>{{ i18n "jakarta-ee-version-10-specifications-list6"  | safeHTML }}</li>
+        <li>{{ i18n "jakarta-ee-version-10-specifications-list7"  | safeHTML }}</li>
+        <li>{{ i18n "jakarta-ee-version-10-specifications-list8"  | safeHTML }}</li>
+      </ul>
+    </div>
+  </div>
+
+
+</div>
+
+<div class="jakartaee-9-features-and-benefits row vertical-align flex-column">
+  <div class="polygons vertical-align"><img width="50" class="icon" src="/release/9/images/icon4.svg" aria-hidden="true"></div>
+  {{ i18n "jakarta-ee-version-9_1-features-participation-title"  | safeHTML }}
+</div>
+
+<div class="container">
+  <div class="card-list contributor-list" data-version="10"></div>
+</div>
+
+<div class="container margin-y-30">
+  <div class="row d-flex flex-wrap justify-content-center">
+    <div class="col-md-8 col-xs-12 vertical-align">
+      <div class="text-center">
+        <h3 class="font-100 text-secondary fw-400 font-exo"><em>29</em></h3>
+        <h4 class="text-blue fw-600 uppercase">{{ i18n "jakarta-ee-version-9_1-participation-projects" }}</h4>
+        <hr class="jakarta-ee-hr" />
+      </div>
+    </div>
+      <div class="col-md-8 col-xs-12 vertical-align">
+          <div class="text-center">
+              <h3 class="font-100 text-secondary fw-400 font-exo"><em>40</em></h3>
+              <h4 class="text-blue fw-600 uppercase">{{ i18n "jakarta-ee-version-9_1-participation-specifications" }}</h4>
+              <hr class="jakarta-ee-hr" />
+          </div>
+      </div>
+
+      <div class="col-md-8 col-xs-12 vertical-align">
+        <div class="text-center">
+            <h3 class="font-100 text-secondary fw-400 font-exo"><em>129</em></h3>
+            <h4 class="text-blue fw-600 uppercase">{{ i18n "jakarta-ee-version-9_1-participation-repositories" }}</h4>
+            <hr class="jakarta-ee-hr" />
+        </div>
+    </div>
+
+    <div class="col-md-8 col-xs-12 col-sm-offset-6 col-md-offset-0 vertical-align">
```

Comment:
![image](https://user-images.githubusercontent.com/6061000/166833804-e796e58f-2647-4906-830d-3ad44aa19509.png)
Issue still present

Reply:
If you remove col-offsets (col-sm-offset-6) issue is fixed and it works fine for all sizes.

Reply:
I updated the PR to remove the offset classes, which as you said, are not needed.